src/mlpack/core/data/string_encoding_impl.hpp

```diff
@@ -0,0 +1,208 @@
+/**
+ * @file string_encoding_impl.hpp
+ * @author Jeffin Sam
```

Comment:
I think this was a team effort, so maybe it would be fair to list @lozhnikov as author as well?

Reply:
I agree.